vite.config.ts

```diff
@@ -29,4 +29,7 @@ export default defineConfig({
             '@': path.resolve(__dirname, './src'),
         },
     },
+    server: {
+        port: '8443',
+    },
```

Comment:
`non-blocking`: whats this for?

Reply:
oh no this is not needed. I'll remove it.